x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -196,17 +197,23 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
                         : [primaryTimestamp],
                       include_unmapped: true,
                       runtime_mappings: runtimeMappings,
+                      ignore_unavailable: true,
```

Comment:
This is the line that allows privilege checking to return results even with a concrete index (no wildcard pattern) that doesn't exist specified in the list of indices/patterns. Previously, any concrete index that doesn't exist being specified would break privilege checking, even if there are other indices/patterns specified in the list as well.